tests/contrib/operators/test_dataproc_operator.py

```diff
@@ -541,18 +592,56 @@ def test_delete_cluster(self):
                 requestId=mock.ANY)
             hook.wait.assert_called_once_with(self.operation)
 
+    def test_render_template(self):
+        task = DataprocClusterDeleteOperator(
+            task_id=TASK_ID,
+            cluster_name=CLUSTER_NAME_TEMPLATED,
+            project_id=GCP_PROJECT_TEMPLATED,
+            region=GCP_REGION_TEMPLATED,
+            dag=self.dag,
+        )
+
+        self.assertEqual(task.template_fields,
+                         ['cluster_name', 'project_id', 'region'])
```

Comment:
I think it would be better to test if elements of `template_fields` are attributes of instance. Then changing attribute name without changing template field would result in test failure. WDYT?

Reply:
`ti.render_template()` should take care of that case. It would fail if it doesn't fiend an attribute listed in template_fields